deflatebench.py

```diff
@@ -604,6 +609,9 @@ def main():
             sys.exit(1)
         cfgRuns['testtool'] = 'minideflate'
 
+    if args.testtool:
+        cfgRuns['testtool'] = args.testtool
```

Comment:
Does this even work?

The next test checks whether it is 'minigzip' or 'minideflate', and does not accept any path.
Also this conflicts with the above two options --minigzip and --minideflate.

In my case, I just put deflatebench homedir into PATH, and run it from the folder where minigzip/minideflate exists.

Reply:
I have rebased, so it might make more sense now how it works. It detects minigzip versus minideflate based on if it exists in the tool name (which is from a previous commit).